portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -20,25 +20,33 @@
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
+import com.google.sps.data.User;
 import com.google.gson.Gson;
 import java.util.List;
 
 /** Servlet that returns some example content. TODO: modify this file to handle comments data */
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
-  
+  ArrayList<User> users = new ArrayList<User>();
```

Comment:
A couple things on this declaration:

1. It's common practice to use type inference in Java now (I think as of Java 7?), so you can write ArrayList<User> users = new ArrayList<>(), and the compiler will fill in the brackets itself.
2. Remember to use the interface List here
3. When making an object field, you want to declare it outside the constructor, but initialize it *inside* the constructor. So you'd have "List<User> users;", and then "users = new ArrayList<User>();" in the constructor.
4. You need to write what classes have access this field. The default for java is 'package-only', which means that every class in the package could call DataServlet.users and potentially modify it. What you want here is 'private', so that the field is only accessible inside the class itself. You'll almost always want your fields set to private.

Reply:
I implemented 1, 2, and 4. Later on I don't use a ArrayList to store the information. Declaring the arraylist in the doGet  function resets the array every everytime. So it wouldn't store any of the comments I posted.